lbz/dev/server.py

```diff
@@ -5,22 +5,26 @@
 import json
 import logging
 import urllib.parse
+from abc import ABCMeta, abstractmethod
 from http.server import BaseHTTPRequestHandler
 from http.server import HTTPServer
-from typing import Tuple, Union
+from typing import Tuple, Union, Type
 
 from lbz.dev.misc import Event
 from lbz.resource import Resource
-from lbz.response import Response
 
 
-class MyLambdaDevHandler(BaseHTTPRequestHandler):
+class MyLambdaDevHandler(BaseHTTPRequestHandler, metaclass=ABCMeta):
     """
     Mimics AWS Lambda behavior.
     """
 
-    cls = None
-    done = False
+    done: bool = False
```

Comment:
I wrote it in a different place but in general, having class attributes that are changed during the life of an instance is risky - however, it can be improved in the future.